PublicationsRouter/v2/sword-out/XWALK-DSpace.md

```diff
@@ -0,0 +1,115 @@
+# JPER Core Metadata to Dublin Core for DSPACE
+
+This document describes the crosswalk from the notification metadata fields received via the JPER API to the
+ XML format supplied to repositories via SWORDv2.
+ 
+For information about the schema see:
+
+* [Dubin Core](http://dublincore.org/documents/dcmi-terms/)
+
```

Comment:
https://wiki.duraspace.org/display/DSDOC4x/Metadata+and+Bitstream+Format+Registries  consider using this link also  

(this is a reminder to myself and Karmele)